test/config/integration/server_http2.json

```diff
@@ -18,10 +18,21 @@
         "access_log": [
         {
           "path": "/dev/null",
-          "filters": [
-            {"type": "status_code", "op": ">=", "value": 500},
-            {"type": "duration", "op": ">=", "value": 1000000}
-          ]
+          "filter" : {
+            "type": "logical_or",
+            "filters": [
+              {
+                "type": "status_code",
+                "op": ">=",
```

Comment:
can you use '=' also in integration test configs, so that it'll have additional coverage